### PR TITLE
Foreman 2.3 settings file

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

**Review by evgeni — CHANGES_REQUESTED** [review]
please adjust `settings` too :+1: 
https://github.com/theforeman/theforeman-rel-eng/blob/8ad94c3180a4773cfd8941fde04c528f479ae1da/settings#L1

**Comment by pcreech** [comment]
adjusted

File: settings [hunk 1]
```diff
@@ -51,7 +51,7 @@ DOCROOT="/var/www/vhosts/downloads/htdocs"
 PASS_NAME_GPG="theforeman/releases/foreman/$FOREMAN_VERSION-gpg"
 PASS_NAME_KEY="theforeman/releases/foreman/$FOREMAN_VERSION-key"
 XARGS_JOBS="-n 20 -P 4"
-GIT_DIR="$HOME/dev"
+GIT_DIR="$HOME/devel/theforeman"
```
Comment (evgeni) [inline settings:54]: nop

Comment (pcreech) [reply settings]: fixed.

**Review by evgeni — APPROVED** [review]

**Pull request merged**